apps/authentication-service/src/init.ts

```diff
@@ -1,10 +1,16 @@
-import { clientsRoutes } from "@/resources/clients/clientsRoutes";
 import cors from "cors";
-import express, { Application, NextFunction, Request, Response } from "express";
 import helmet from "helmet";
 import nocache from "nocache";
+import { rateLimit } from "express-rate-limit";
+import express, { Application, NextFunction, Request, Response } from "express";
+
 import { errorHandler } from "@/middleware/errorMiddleware";
 import { notFoundHandler } from "@/middleware/notFoundMiddleware";
+import { clientsRoutes } from "@/resources/clients/clientsRoutes";
+
+const rateLimitWindowMinutes = 10;
+
+const isTesting = () => process.env.ENV === "test" || process.env.IS_GITHUB;
```

Comment:
Added method to tell whether we're in testing mode or not. This was especially necessary for GH CI/CD to work. 

The ENV var on GH is set to `prod`, which seems necessary for some tests, because, if I changed it to `test`, some of them broke. 

So, I added a GH VAR so I could detect when it's running on GH.

Reply:
Interesting, in ucw-app the env var is set to test. Doesn't it make sense to have the github environment to be "test" since it's literally a test environment?

Reply:
I've been thinking about this. I could see having GH set to "prod", since we want the tests to mimic prod, but right now, at least in the auth service for the rate-limiting test, the different ENV values don't change anything.